app/models/like.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: likes
+#
+#  id           :bigint           not null, primary key
+#  likable_type :string
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#  likable_id   :bigint
+#  user_id      :bigint
+#
+# Indexes
+#
+#  index_likes_on_likable  (likable_type,likable_id)
+#  index_likes_on_user_id  (user_id)
+#
+class Like < ApplicationRecord
+  belongs_to :likable, polymorphic: true
```

Comment:
`  belongs_to :likeable, polymorphic: true`

Reply:
I suggest this variant, such as 'imageable in this guide'
https://guides.rubyonrails.org/association_basics.html#polymorphic-associations